### PR TITLE
Dockerize local

### PR DESCRIPTION
**Pull request by alexanderward**
This PR is to allow developers the ability to run the application in Docker. 

 Directions are in deployments/local/README.md

### CODE REVIEW

File: deployment/local/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+mkdir -p local
+
+if [ -f "local/first_run.lock" ]; then
+    echo "[ + ] Execution of first_run previously completed.  Skipping."
+else
+    echo "[ + ] Executing first_run"
+    python3 install.py --non-interactive --docker
+    touch local/first_run.lock
+fi
```
Comment (davidpofo) [inline deployment/local/docker-entrypoint.sh:9]:
```suggestion

```

Comment (alexanderward) [reply deployment/local/docker-entrypoint.sh:9]: if statements need to be closed in bash.

govready_q    | /usr/src/app/deployment/local/docker-entrypoint.sh: line 15: syntax error: unexpected end of file

**Comment by davidpofo** [comment]
Ah right I had my python scripting glasses on.

**Comment by kerryrm** [comment]
@alexanderward I'm excited to try this out, but I'm a bad choice as a reviewer of docker related things. :)

**Pull request merged**